pom.xml

```diff
@@ -14,7 +14,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.16.RELEASE</version>
+        <version>2.3.0.RELEASE</version>
```

Comment:
Oh interesting... this must be why I couldn't get it to work. I was changing the version of the hateoas dependency and didn't see any changes (as you mentioned in Teams, it was pulling down an old version of the library despite my changes to the dependency directly).